src/openrct2/actions/FootpathPlaceAction.hpp

```diff
@@ -131,7 +127,7 @@ DEFINE_GAME_ACTION(FootpathPlaceAction, GAME_COMMAND_PLACE_PATH, GameActionResul
             {
                 // It is possible, let's remove walls between the old and new piece of path
                 auto zLow = _loc.z;
-                auto zHigh = zLow + (4 * 8);
+                auto zHigh = zLow + PATH_HEIGHT;
```

Comment:
Not a big fan of this name for the constant. Clearance height might be better. Although I realise this isn't part of the PR.